markdown/ddl/ddl-database.html.md.erb

```diff
@@ -2,77 +2,75 @@
+HAWQ uses two other database templates internally, `template0` and `postgres`. Do not drop or modify `template0` or `postgres`. You can use `template0` to create a completely clean database containing only the standard objects predefined by HAWQ at initialization, especially if you have modified `template1`.
 
 ## <a id="topic4"></a>Creating a Database 
 
-The `CREATE DATABASE` command creates a new database. For example:
+You must have appropriate privileges or be a HAWQ superuser to create a database. If you do not have the correct privileges, you cannot create a database. The HAWQ administrator must either grant you the necessary privileges or create a database for you.
+
+The [CREATE DATABASE](../reference/sql/CREATE-DATABASE.html) command creates a new database. For example:
 
 ``` sql
-=> CREATE DATABASE new_dbname;
+=# CREATE DATABASE <new_dbname>;
 ```
 
-To create a database, you must have privileges to create a database or be a HAWQ superuser. If you do not have the correct privileges, you cannot create a database. The HAWQ administrator must either give you the necessary privileges or to create a database for you.
-
-You can also use the client program `createdb` to create a database. For example, running the following command in a command line terminal connects to HAWQ using the provided host name and port and creates a database named *mydatabase*:
+You can also use the client program `createdb` to create a database. For example, running the following command connects to HAWQ on the host named `hmaster` at port `5432` and creates a database named `mydatabase`:
 
 ``` shell
-$ createdb -h masterhost -p 5432 mydatabase
+$ createdb -h hmaster -p 5432 mydatabase
 ```
 
-The host name and port must match the host name and port of the installed HAWQ system.
+The host name and port must match the host name and port of the HAWQ master node.
 
-Some objects, such as roles, are shared by all the databases in a HAWQ system. Other objects, such as tables that you create, are known only in the database in which you create them.
+Some objects, such as roles, are shared by all of the databases in a HAWQ deployment. Other objects, such as tables that you create, are known only to the database in which you create them.
 
 ### <a id="topic5"></a>Cloning a Database 
 
-By default, a new database is created by cloning the standard system database template, *template1*. Any database can be used as a template when creating a new database, thereby providing the capability to 'clone' or copy an existing database and all objects and data within that database. For example:
+By default, a new database is created by cloning the standard system database template, `template1`. You may identify an alternate template to use for database creation. As mentioned in the section above, you can specify `template0` as the template:
```

Comment:
Also re: cloning, the CREATE DATABASE page has this wording, which seems to suggest a regular db can be named as a template.  Probably want to do some editing here as well:

"When you copy a database by specifying its name as the template, no other sessions can be connected to the template database while it is being copied. New connections to the template database are locked out until CREATE DATABASE completes."